platform/security/security-oidc-realm/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -23,6 +23,7 @@
         <cm:managed-properties persistent-id="org.codice.ddf.security.oidc.realm.OidcRealm"
                                update-strategy="container-managed"/>
         <property name="oidcHandlerConfiguration" ref="oidcHandlerConfiguration"/>
+        <property name="usernameAttributeList" value="preferred_username, name, email, sub"/>
```

Comment:
📘 Added defaults here because they weren't getting picked up. Identical to the way we do it in SAML.